.github/workflows/be-test-report.yml

```diff
@@ -0,0 +1,41 @@
+name: "Backend Test Report"
+
+on:
+  workflow_run:
+    workflows: ["Check Backend Pull Request"]
+    types:
+      - completed
+
+jobs:
+  be-report:
+    name: Backend test report
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: 📁 Download test results
+        id: download-artifact
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          name: be-test-results
+          path: backend
+          workflow: check-be-pull-request.yml
```

Comment:
Making sure I understand, the `check-be-pull-request`  will be ran and its result on `success` will then be passed on to the next step?

Reply:
Close 😅 

To be accurate: `be-test-report` listens for successful completion of workflow `check-be-pull-request`. (The former does not trigger the latter)

But I guess that's what you're already thinking, just wanted to be accurate 🙇